mulesoft-inference/1.1/modules/ROOT/pages/mulesoft-inference-connector-reference.adoc

```diff
@@ -0,0 +1,2489 @@
+| Top P a| Number |  Controls diversity by creating a nucleus of the most probable words to choose from for the next token. Specifies the cumulative probability score threshold that the tokens must reach. |  0.9 | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+[[TextGenerationConfig_Mistralai]]
+===== Mistral AI
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Proxy Configuration a| One of:
+
+* <<Proxy>>
+* <<NtlmProxy>> | Configures a proxy for outbound connections. |  | 
+| Mistral AI Model Name a| String | Model name. |  | x
+| TLS Configuration a| <<Tls>> |  If HTTPS is configured as a protocol, then you must configure at least the keystore configuration. |  | 
+| API Key a| String |  API key as required by the inference provider. |  | x
+| Custom headers a| Array of <<DefaultHeader>> |  Custom headers are additional HTTP headers to include with each request. For any existing header key, the custom value will take precedence and overwrite the existing one.|  |
+| Timeout a| Number |  Response timeout value set for each inference HTTP request. |  60 | 
+| Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Response timeout unit for the *Timeout* field. |  SECONDS | 
+| Max Tokens a| Number |  Defines the number of LLM tokens to use when generating a response. This field helps control the usage and costs when engaging with LLMs. |  500 | 
+| Temperature a| Number |  Number between 0 and 2. The temperature is used to control the randomness of the output. When you set it higher, you'll get more random outputs. When you set it lower (towards 0), the values are more deterministic. |  0.9 | 
+| Top P a| Number |  Controls diversity by creating a nucleus of the most probable words to choose from for the next token. Specifies the cumulative probability score threshold that the tokens must reach. |  0.9 | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+[[TextGenerationConfig_Nvidia]]
+===== Nvidia
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Proxy Configuration a| One of:
+
+* <<Proxy>>
+* <<NtlmProxy>> | Configures a proxy for outbound connections. |  | 
+| Nvidia Model Name a| String |  Model Name. |  | x
+| [Nvidia] Base URL a| String | Nvidia base URL. |  https://integrate.api.nvidia.com |
```

Comment:
```suggestion
| [Nvidia] Base URL a| String | Nvidia base URL. |  `\https://integrate.api.nvidia.com` |
```

Reply:
We format base URLs like this so the link isn't clickable to users and is instead static